pkg/scheduler/test_scheduler.go

```diff
@@ -247,6 +250,19 @@ func (s *Scheduler) getExecuteOptions(namespace, id string, request testkube.Exe
 		}
 
 		testCR.Spec = mergeContents(testCR.Spec, testSourceCR.Spec)
+
+		if testSourceCR.Spec.Type_ == "" && testSourceCR.Spec.Repository.Type_ == "git" {
+			fetcher := content.NewFetcher(tempDir)
+			content := testsmapper.MapTestContentFromSpec(testCR.Spec.Content)
+			t, err := fetcher.CalculateGitContentType(*content.Repository)
+			if err == nil {
+				testCR.Spec.Content.Type_ = t
+				testCR.Spec.Content.Repository.Type_ = t
```

Comment:
guess, we use `git` for this now

Reply:
removed this line, tested with git-file and git-dir, still works